rebar.config.script

```diff
@@ -87,13 +87,18 @@ MakeDep = fun
         {AppName, ".*", {git, Url, Version}, Options}
 end,
 
+ErlOpts = case os:getenv("ERL_OPTS") of
+    "bin_opt_info" -> [bin_opt_info, debug_info, {i, "../"}];
+    Else -> [Else, {i, "../"}]
```

Comment:
I think we want a clause here for false if ERL_OPTS is not found otherwise it will add a false to the list